config/templates/org-member-added.html

```
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html lang="en"><head><title></title></head>
```

HTMLHint (1.9.2)

[error] <title></title> must not be empty.

(title-require)